src/applications/vaos/actions/newAppointment.js

```diff
@@ -278,7 +280,10 @@ export function openFacilityPage(page, uiSchema, schema) {
       // If we have the VA parent in our state, we don't need to
       // fetch them again
       if (!parentFacilities) {
-        parentFacilities = await getOrganizations(userSystemIds);
+        parentFacilities = await getOrganizations({
+          siteIds: userSiteIds,
+          useVSP,
```

Comment:
Worth noting that when this is on, all subsequent calls will fail, because the real apis don't return `varXYZ` ids. This is ok, I don't expect the flow to work until all calls are updated to switch between VSP and var-resources.